### PR TITLE
Add script to run `super-linter` locally

### LINKED ISSUE #98: Add script to run `super-linter` locally
During local development is hard to write code following `super-linter` rules and for each mistake, it will be necessary to run the corresponding action on GitHub, which leads to a waste of time.

Because of this, we need a method to be able to run the `super-linter` locally.

My suggestion would be to add a script to check this inside `util` folder. You can see [here] an example to run `super-linter` locally.

[here]: https://github.com/github/super-linter/blob/main/docs/run-linter-locally.md

### PR DESCRIPTION
**Pull request by gabrielmocanu**
- delete deprecated `update-repo.sh`
- Add script with parameters to be able to run linter locally

Closes #98.

### CODE REVIEW

**Review by teodutu — CHANGES_REQUESTED** [review]
I tested the script and it works fine. It's annoying that it doesn't account for global configs such as [`CPPLINT.cfg`](https://github.com/open-education-hub/operating-systems/blob/master/content/CPPLINT.cfg) when running on a certain file alone. Could you make it aware of such configs? Other than this, see my inline comments.

File: util/super-linter.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# PATH_TO_CODEBASE: The path to the codebase you want to lint. Don't use relative paths.
+# PATH_TO_LINT_RULES: The path to the lint rules you want to use. Don't use relative paths.
```
[the remark this answers is not in the document]
Comment (gabrielmocanu) [reply util/super-linter.sh:4]: Relative paths don't work with bind mounts. But yup, we can use also `realpath`.

**Comment by gabrielmocanu** [comment]
@teodutu 

Unfortunately, this tool is made in hell.
It is very related to github actions, that's why you also saw those strange paths for `MARKDOWN_CONFIG_FILE` and `MARKDOWN_CUSTOM_RULE_GLOBS`.
There is no config to specify where to look for `CPPLINT.cfg`.

**Review by teodutu — APPROVED** [review]
I saw your comment about the tight coupling between `super-linter` and GitHub actions. It's sad, but we'll leave it as it is.

**Pull request merged**